app/helpers/images_helper.rb

```diff
@@ -0,0 +1,23 @@
+module ImagesHelper
+  def bg_img(src, alt, css_class: 'c-bg-img', extra_css_class: nil)
+    tag.div("", alt: alt,
+                role: "img",
+                class: [css_class, extra_css_class].compact.join(" "),
+                style: "background-image:url(#{src})")
+  end
+
+  def rounded_bg_img(src, alt, css_class: 'c-rounded-bg-img', extra_css_class: nil)
+    bg_img(src, alt, css_class: css_class, extra_css_class: extra_css_class)
+  end
+
+  def rounded_bg_link(src, href, title, alt, css_class)
+    link_to(
+      href,
+      class: ['c-rounded-bg-img', css_class].compact.join(" "),
+      style: "background-image:url(#{src})",
+      "aria-title": title
+    ) do
+      image_tag(src, alt: alt, class: "tw-sr-only")
+    end
+  end
+end
```

Comment:
@SleeplessByte Please give this a check.

Reply:
Looks good. One comment, see inline.

You might want to make them keyword args so you don't need to lookup the order all the time. But that's just me :)

Reply:
Yeah - good suggestion. I'll see how much it pisses me off looking it up :)